cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Date extraction</title>
+</head>
+<body>
+<div id ="help">
+	<h4>General explanation</h4>
+	<p>
+The date extraction function enables user to extract a date from a string type of the source data and assign the value to the target source.<br>
+ </br>
+ <br/>
+For the correct application, the date format has to be specified by the user. In the example below the date is specified by month (MM), 
```

Comment:
The date format must be specified by the user to acheive the correct result.

Reply:
achieve?